packages/server/tsconfig.json

```diff
@@ -16,14 +15,15 @@
     "noUnusedParameters": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    
     "moduleResolution": "node",
     "skipLibCheck": true,
     "typeRoots": [
-      "node_modules/@types",
-      "../../node_modules/@types"
+      "../../node_modules/@types",
```

Comment:
the reason this was in here was because I believed that dev deps of a package got installed in the local node_modules of that package, and not the workspace's. 

I could be wrong, in which case you can remove it, but I'd check first.

Reply:
There don't seem to be any "@types"  getting installed into the server node_modules (though some of the other folders do get installed here... no idea why...)

Reply:
hmmm peculiar